spec/how.md

```diff
@@ -1,7 +1,7 @@
 # `vt6/how` - Design overview
 
 This document is non-normative.
-For the formal definitions of basic VT6 terminology, please refer to [`vt6/core`](../core/).
+For the formal definitions of basic VT6 terminology, please refer to [`vt6/core`](core/).
```

Comment:
This fixes the links on Github, but breaks them on vt6.io, so I'm not going to merge this.